packages/cli/src/License.ts

```diff
@@ -357,6 +357,12 @@ export class License {
 		return this.getFeatureValue(LICENSE_QUOTAS.TEAM_PROJECT_LIMIT) ?? 0;
 	}
 
+	getConcurrencyProductionLimit() {
+		return (
+			this.getFeatureValue(LICENSE_QUOTAS.CONCURRENCY_PRODUCTION_LIMIT) ?? UNLIMITED_LICENSE_QUOTA
```

Comment:
I don't think we should be limiting concurrency via license.

That's a product discussion but I'd say we should toggle on or off the feature via license, but the number of concurrent executions may be handled by an environment variable right? Just making sure we have the right specs here.

Reply:
It's part of the reqs, please see here: https://github.com/n8n-io/n8n/pull/9453#discussion_r1629094580

Reply:
See [2351580](https://github.com/n8n-io/n8n/pull/9453/commits/235158063a1e38a9377048f38067df62004af45d)